modules/nw-external-dns-operator-logs.adoc

```diff
@@ -18,9 +18,9 @@ $ oc logs -n external-dns-operator deployment/external-dns-operator -c external-
 
 == External DNS Operator domain name limitations
 
-External DNS Operator uses the TXT registry, which follows the new format and adds the prefix for the TXT records. This reduces the maximum length of the domain name for the TXT records. A DNS record cannot be present without a corresponding TXT record, so the domain name of the DNS record must follow the same limit as the TXT records. For example, DNS record is `<domain-name-from-source>` and the TXT record is `external-dns-<record-type>-<domain-name-from-source>`.
+External DNS Operator uses the TXT registry, which adds the prefix for the TXT records. This reduces the maximum length of the domain name for the TXT records. A DNS record cannot be present without a corresponding TXT record, so the domain name of the DNS record must follow the same limit as the TXT records. For example, DNS record is `<domain-name-from-source>` and the TXT record is `external-dns-<record-type>-<domain-name-from-source>`.
```

Comment:
"The External DNS Operator uses a TXT registry which adds a prefix for TXT records. This reduces the maximum length of the domain name for TXT records. A DNS record cannot be present without a corresponding TXT record, so the domain name of the DNS record must follow the same limit as the TXT records. For example, a DNS record of `<domain-name-from-source>` results in a TXT record of `external-dns-<record-type>-<domain-name-from-source>`."

Reply:
Is the limit of the DNS record shorter than the TXT record due to the prefix?

Reply:
Yes!